web/src/constants/AttributesTags.ts

```diff
@@ -0,0 +1,275 @@
+  // K8S_CLUSTER_NAME: {tags:[],description:""},
+  // K8S_CONTAINER_NAME: {tags:[],description:""},
+  // K8S_CRONJOB_NAME: {tags:[],description:""},
+  // K8S_CRONJOB_UID: {tags:[],description:""},
+  // K8S_DAEMONSET_NAME: {tags:[],description:""},
+  // K8S_DAEMONSET_UID: {tags:[],description:""},
+  // K8S_DEPLOYMENT_NAME: {tags:[],description:""},
+  // K8S_DEPLOYMENT_UID: {tags:[],description:""},
+  // K8S_JOB_NAME: {tags:[],description:""},
+  // K8S_JOB_UID: {tags:[],description:""},
+  // K8S_NAMESPACE_NAME: {tags:[],description:""},
+  // K8S_NODE_NAME: {tags:[],description:""},
+  // K8S_NODE_UID: {tags:[],description:""},
+  // K8S_POD_NAME: {tags:[],description:""},
+  // K8S_POD_UID: {tags:[],description:""},
+  // K8S_REPLICASET_NAME: {tags:[],description:""},
+  // K8S_REPLICASET_UID: {tags:[],description:""},
+  // K8S_STATEFULSET_NAME: {tags:[],description:""},
+  // K8S_STATEFULSET_UID: {tags:[],description:""},
+  // KIND: {tags:[],description:""},
+  // MESSAGE_COMPRESSED_SIZE: {tags:[],description:""},
+  // MESSAGE_ID: {tags:[],description:""},
+  // MESSAGE_TYPE: {tags:[],description:""},
+  // MESSAGE_UNCOMPRESSED_SIZE: {tags:[],description:""},
+  // MESSAGING_CONSUMER_ID: {tags:[],description:""},
+  // MESSAGING_CONVERSATION_ID: {tags:[],description:""},
+  // MESSAGING_DESTINATION: {tags:[],description:""},
+  // MESSAGING_DESTINATION_KIND: {tags:[],description:""},
+  // MESSAGING_KAFKA_CLIENT_ID: {tags:[],description:""},
+  // MESSAGING_KAFKA_CONSUMER_GROUP: {tags:[],description:""},
+  // MESSAGING_KAFKA_MESSAGE_KEY: {tags:[],description:""},
+  // MESSAGING_KAFKA_PARTITION: {tags:[],description:""},
+  // MESSAGING_KAFKA_TOMBSTONE: {tags:[],description:""},
+  // MESSAGING_MESSAGE_ID: {tags:[],description:""},
+  // MESSAGING_MESSAGE_PAYLOAD_COMPRESSED_SIZE_BYTES: {tags:[],description:""},
+  // MESSAGING_MESSAGE_PAYLOAD_SIZE_BYTES: {tags:[],description:""},
+  // MESSAGING_OPERATION: {tags:[],description:""},
+  // MESSAGING_PROTOCOL: {tags:[],description:""},
+  // MESSAGING_PROTOCOL_VERSION: {tags:[],description:""},
+  // MESSAGING_RABBITMQ_ROUTING_KEY: {tags:[],description:""},
+  // MESSAGING_SYSTEM: {tags:[],description:""},
+  // MESSAGING_TEMP_DESTINATION: {tags:[],description:""},
+  // MESSAGING_URL: {tags:[],description:""},
+  // NAME: {tags:[],description:""},
+  // NET_HOST_CARRIER_ICC: {tags:[],description:""},
+  // NET_HOST_CARRIER_MCC: {tags:[],description:""},
+  // NET_HOST_CARRIER_MNC: {tags:[],description:""},
+  // NET_HOST_CARRIER_NAME: {tags:[],description:""},
+  // NET_HOST_CONNECTION_SUBTYPE: {tags:[],description:""},
+  // NET_HOST_CONNECTION_TYPE: {tags:[],description:""},
+  // NET_HOST_IP: {tags:[],description:""},
+  // NET_HOST_NAME: {tags:[],description:""},
+  // NET_HOST_PORT: {tags:[],description:""},
+  // NET_PEER_IP: {tags:[],description:""},
+  NET_PEER_NAME: {description: 'Name of the database host.', tags: ['example.com']},
```

Comment:
@xoscar is that what you need?

Reply:
Yeah, something like this, I'd suggest having this as json files. After that we need to add the description to the attribute dropdown.

Reply:
@xoscar where in the attribute dropdown?